docs/troubleshooting/deadline-exceeded-error.mdx

```diff
@@ -40,10 +40,10 @@ If you're using Temporal Cloud, create a [support ticket](/cloud/support#support
 [Frontend Service](/clusters#frontend-service) logs can show which parts of the Cluster aren't working.
 For the error to appear, a service pod or container must be up and running.
 
-OSS users can verify that the Frontend Service is connected and running by using `tctl`.
+OSS users can verify that the Frontend Service is connected and running by using `temporal`.
```

Comment:
We want to refer to it as "Temporal CLI" if we can -

I still see many instances where the conversion of `tctl` resulted in `temporal`.
But I think we want it to convert from `tctl` to "Temporal CLI"

Reply:
They didn't. I specifically used lower-case to reflect the CLI utility. Everything was done by hand one-by-one, so it's not a result of copy/paste.

I see that you prefer using Temporal CLI, so I will stick to that term in the future.